src/views/checkups/CheckupsDetailsPageHistoryRow.tsx

```diff
@@ -25,13 +29,21 @@ const CheckupsNetworkDetailsPageHistoryRow: FC<
         />
       </TableData>
       <TableData activeColumnIDs={activeColumnIDs} id="status">
-        <CheckupsNetworkStatusIcon job={job} onlyJob={true} />
+        <CheckupsStatusIcon job={job} onlyJob={true} />
+      </TableData>
+      <TableData activeColumnIDs={activeColumnIDs} id="start-time">
+        <Timestamp timestamp={job?.status?.startTime || NO_DATA_DASH} />
       </TableData>
       <TableData activeColumnIDs={activeColumnIDs} id="complete-time">
         <Timestamp timestamp={job?.status?.completionTime || NO_DATA_DASH} />
       </TableData>
+      {rowData?.customActions && (
```

Comment:
1. the extra column should be reflected in the column definition (check other tables with actions)
2. since both column definition and row definition should be in sync it's probably easier to add empty (`null`) actions for other types
3.  passing the custom actions as render prop (backed by useCallback) brings no advantage over passing it as a component function i.e.  as we do with the row `Row={CheckupsDetailsPageHistoryRow}`.